public_officials/tests.py

```diff
@@ -3,6 +3,7 @@
 from django.core.urlresolvers import resolve
 from public_officials.views import *
 from public_officials.models import *
+from django.contrib.auth.models import User
```

Comment:
@andrewdwooten is this bringing in the auth user model? Is this the same as our user model? Thanks for explaining 🏅 🤴